cilantro_audit/view_audit_templates.py

```diff
@@ -0,0 +1,50 @@
+import kivy
```

Comment:
Needs the auto formatting for PEP-8 consistency.
See https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html

Reply:
Done.